src/docker-existing-docker-compose/.devcontainer/docker-compose.yml

```diff
@@ -13,7 +13,7 @@ services:
 
     volumes:
       # Update this to wherever you want VS Code to mount the folder of your project
-      - ../..:/workspaces:cached
+      - ..:/workspaces:cached
```

Comment:
Can we bump the patch version here, so that these changes are reflected in the UX? 

https://github.com/devcontainers/templates/blob/a98e68aa28930ad3e0b78c8ffe638ba2d91f9e92/src/docker-existing-docker-compose/devcontainer-template.json#L3